starport/pkg/localfs/search.go

```diff
@@ -1,18 +1,56 @@
 package localfs
 
 import (
+	"os"
+	"path/filepath"
 	"sort"
-
-	"github.com/mattn/go-zglob"
+	"strings"
 )
 
 // Search searches for files in the fs with given glob pattern by ensuring that
 // returned file paths are sorted.
-func Search(pattern string) (paths []string, err error) {
-	files, err := zglob.Glob(pattern)
+func Search(path, pattern string) (paths []string, err error) {
```

Comment:
Search does not have any logic other than calling `glob`, can we move contents of glob here?

Reply:
yes, I agree!!